### ISSUE TITLE
Custom global playlist

### ISSUE THREAD

**Issue by jizquizzo** [vincelwt/harmony#82]
Would be nice to have the option to create local playlist with the abilities of mixing local library tracks and services tracks.

**Comment by vincelwt** [comment]
Agree, but first I think we need to implement m3u playlists. I think this will be available around 0.4 or 0.5

**Comment by gcornut** [comment]
Would this mean we could create a playlist with the same name on multiple music services and have them appear as merged into one playlist under harmony?

**Comment by vincelwt** [comment]
@gcornut Actually I was more thinking about creating a local playlist referring to multiple songs from different services. 

But I like your idea, didn't thought of it ! I'll think about it
